content/en/news/releases/1.13.x/announcing-1.13/_index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Announcing Istio 1.12
```

Comment:
A number of lines have 1.12 in the header.

Reply:
```suggestion
title: Announcing Istio 1.13
```